.vitepress/config.ts

```diff
@@ -176,6 +176,45 @@ export default defineConfig({
             }
           ]
         }
+      ],
+      '/config/': [
+        {
+          text: '設定',
+          items: [
+            {　　　　　　　　　　
+              text: 'Vite の設定',
+              link: '/config/'
+            },
+            {
+              text: '共有オプション',
+              link: '/config/shared-options'
+            },
+            {
+              text: 'サーバーオプション',
+              link: '/config/server-options'
+            },
+            {
+              text: 'ビルドオプション',
+              link: '/config/build-options'
+            },
+            {
+              text: 'プレビューオプション',
```

Comment:
（これはページのタイトルを変えてもいいかも知れない）
```suggestion
              text: 'プレビューのオプション',
```

Reply:
修正しました。